File: 4316-minimum-swaps-to-move-zeros-to-end/minimum-swaps-to-move-zeros-to-end.py
class Solution:
    def minimumSwaps(self, nums: list[int]) -> int:
        n = len(nums)
        zero = nums.count(0)
        swap = 0

        for i in range(n - zero):
            if nums[i] == 0:
                swap += 1

        return swap

File: 4316-minimum-swaps-to-move-zeros-to-end/minimum-swaps-to-move-zeros-to-end.go
func minimumSwaps(nums []int) int {
	n := len(nums)
	zero, swap := 0, 0

	for _, num := range nums {
		if num == 0 {
			zero++
		}
	}

	for i := 0; i < n-zero; i++ {
		if nums[i] == 0 {
			swap++
		}
	}

	return swap
}